app/jobs/remote_server_job.rb

```diff
@@ -20,53 +33,47 @@ def perform(initiator_key, benchmark, options = {})
   private
 
   def ruby_trunk(ssh, commit_hash, options)
-    options.reverse_merge!({
-      ruby_benchmarks: true,
-      ruby_memory_benchmarks: true,
-      optcarrot_benchmarks: true,
-      liquid_benchmarks: true
-    })
-
-    execute_ssh_commands(ssh,
-      [
-        "docker pull rubybench/ruby_trunk",
-        "docker run --rm
-          -e \"RUBY_BENCHMARKS=#{options[:ruby_benchmarks]}\"
-          -e \"RUBY_MEMORY_BENCHMARKS=#{options[:ruby_memory_benchmarks]}\"
-          -e \"OPTCARROT_BENCHMARK=#{options[:optcarrot_benchmarks]}\"
-          -e \"LIQUID_BENCHMARK=#{options[:liquid_benchmarks]}\"
-          -e \"RUBY_COMMIT_HASH=#{commit_hash}\"
-          -e \"API_NAME=#{Rails.application.secrets.api_name}\"
-          -e \"API_PASSWORD=#{Rails.application.secrets.api_password}\"
-          #{build_include_patterns(options[:include_patterns])}
-          rubybench/ruby_trunk".squish
-      ]
-    )
+    ruby = true
```

Comment:
yeah I guess. You have script which takes arguments but on the other side you are calling it always with the same, currently.

I moved those values to variables so it could be somehow easier to tell which value stands for which argument.